### ISSUE TITLE
Is Celo able to map addresses to email and identities other than phone number?

### ISSUE THREAD

**Issue by johnnynanjiang** [celo-org/docs#1402]

**Comment by therealharpaljadeja** [comment]
Yes, you can use any identifier of your choice. However, the SDK has default support for some this allows composability among issuers.

https://github.com/celo-org/social-connect/blob/8b16f84cd7f8088e417c0b090042e86cb0230aa1/packages/odis-identifiers/src/identifier.ts#L25-L36

**Comment by palango** [comment]
@johnnynanjiang Closing this issue, but feel free to contact us again with any further questions.
